### DISCUSSION TITLE
Bundle static export next into single html file

### DISCUSSION THREAD

**Discussion by radenhorcmcb** [vercel/next.js#62043] [Help]
### Summary

Is there any way to bundle next static export into single html file for local use.

**Comment by mutasim77** [comment]
Yes, it's actually possible, but not optimal or recommended. You can achieve that by doing some webpack configuration. Here, you can check: ["All in one file with webpack"](https://stackoverflow.com/questions/44839837/how-to-make-all-in-one-file-with-webpack)

**Reply by radenhorcmcb** [reply]
But there are anyway to open index.html after next static export work without starting a local web server
